test/PushNotificationsTests/APITests.cpp

```diff
@@ -4,32 +4,21 @@
 #include "pch.h"
 #include <testdef.h>
 #include "Shared.h"
-#include "winrt/Windows.Foundation.h"
 
 using namespace WEX::Common;
```

Comment:
Can you also please fix the bug in IsPackagedProcess() in this review. It incorrectly assumes that the error code is an HRESULT